prometheus_juju_backup_all_exporter/utils.py

```diff
@@ -0,0 +1,130 @@
+import json
+from logging import getLogger
+from pathlib import Path
+
+logger = getLogger(__name__)
+
+
+def get_result_code_name(result_code):
+    result_code = int(result_code)
+    status_name = {
+        0: "StatusOK",
+        1: "StatusWarning",
+        2: "StatusCritical",
+        3: "StatusUnknown",
+    }
+    return status_name.get(result_code, "InvalidResultCode")
+
+
+class BackupStats:
+    """A class representing backup statistic file."""
+
+    def __init__(self, config):
+        """Initialize class."""
+        self.set_properties(Path(config.backup_path, "backup_stats.json"))
+
+    @property
```

Comment:
I think "read-only" is suitable in this case since I try to represent the file with a class. But yes I agree that we should put `self._xxx` in the `__init__()`, otherwise it might be confusing where it comes from. I've simplified `utils.py` a bit, in particular it makes not sense to use `_set_properies()` and `_set_default_properies()`.